dspace-spring-rest/src/test/java/org/dspace/app/rest/CommunityRestRepositoryIT.java

```diff
@@ -26,6 +36,65 @@
 
 public class CommunityRestRepositoryIT extends AbstractControllerIntegrationTest {
 
+    public void createTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        ObjectMapper mapper = new ObjectMapper();
+        CommunityRest comm = new CommunityRest();
+        comm.setName("Test Top-Level Community");
+
+        MetadataEntryRest description = new MetadataEntryRest();
+        description.setKey("dc.description");
+        description.setValue("<p>Some cool HTML code here</p>");
+
+        MetadataEntryRest abs = new MetadataEntryRest();
+        abs.setKey("dc.description.abstract");
+        abs.setValue("Sample top-level community created via the REST API");
+
+        MetadataEntryRest contents = new MetadataEntryRest();
+        contents.setKey("dc.description.tableofcontents");
+        contents.setValue("<p>HTML News</p>");
+
+        MetadataEntryRest copyright = new MetadataEntryRest();
+        copyright.setKey("dc.rights");
+        copyright.setValue("Custom Copyright Text");
+
+        MetadataEntryRest title = new MetadataEntryRest();
+        title.setKey("dc.title");
+        title.setValue("Title Text");
+
+        comm.setMetadata(Arrays.asList(description,
+                                       abs,
+                                       contents,
+                                       copyright,
+                                       title));
+
+        String authToken = getAuthToken(admin.getEmail(), password);
+        getClient(authToken).perform(post("/api/core/communities")
```

Comment:
This test looks good. We should also add a test which shows you cannot POST a new community if you don't have rights to do so (or are not logged in).

Reply:
Will do.